connectors/sources/jira.py

```diff
@@ -320,8 +371,310 @@ def get_default_configuration(cls):
+                                access_control = await anext(
+                                    self._project_role_members(
+                                        project=project,
+                                        role_id=role_id,
+                                        access_control=access_control,
+                                    )
+                                )
+            else:
+                self._logger.debug(
+                    f"Issue security level is not set for an issue: {issue_key}. Hence, Assigning project permissions"
+                )
+                project_access_controls = await self._project_access_control(
+                    project=project
+                )
+                return project_access_controls
+        return list(access_control)
+
+    async def _user_access_control_doc(self, user):
+        """Constructs a user access control document, which will be synced to the corresponding access control index.
+        The `_id` of the user access control document will either be the username (can also be the email sometimes) or the email itself.
+        Note: the `_id` field won't be prefixed with the corresponding identity prefix ("user" or "email").
+        The document contains all groups of a user and his email and/or username under `query.template.params.access_control`.
+
+        Returns:
+            dict: dictionary representing an user access control document
+            {
+                "_id": "some.user@spo.com",
+                "identity": {
+                    "email": "email:some.user@spo.com",
+                    "username": "user:some.user",
+                    "user_id": "user_id:some user id"
+                },
+                "created_at": "2023-06-30 12:00:00",
+                "query": {
+                    "template": {
+                        "params": {
+                            "access_control": [
+                                "user:some.user",
+                                "group:1234-abcd-id"
+                            ]
+                        }
+                    }
+                }
+            }
+        """
+        account_id = user.get("accountId")
+        account_name = user.get("displayName")
+
+        _prefixed_account_id = _prefix_account_id(user_id=account_id)
+        _prefixed_user_name = _prefix_username(user=account_name)
+
+        _prefixed_group_ids = {
+            _prefix_group_id(group_id=group.get("groupId", ""))
+            for group in user.get("groups", {}).get("items", [])
+        }
+        _prefixed_role_keys = {
+            _prefix_role_key(role_key=role.get("key", ""))
+            for role in user.get("applicationRoles", {}).get("items", [])
+        }
+
+        user_document = {
+            "_id": account_id,
+            "identity": {
+                "account_id": _prefixed_account_id,
+                "display_name": _prefixed_user_name,
+            },
+            "created_at": iso_utc(),
```

Comment:
question: is there no `creationDate` field or something similar returned from the Jira API?

Reply:
No, there's no `creationDate` or similar field getting from the Jira API response.